.github/workflows/update-bpmn-visualization-version.yml

```diff
@@ -15,33 +15,33 @@ jobs:
       VERSION: ${{ github.event.client_payload.version || github.event.inputs.version }}
     steps:
       - uses: actions/checkout@v3
-      - name: Get old bpmn-visualization version
+      - name: Get the old version of bpmn-visualization
         id: lookupOldBPMNVisuVersion
         uses: mikefarah/yq@v4.30.5
         with:
-          cmd: yq '.dependencies[] | select(.name == "bpmn-visualization") | .version' inst/htmlwidgets/bpmnVisualization.yaml
+          cmd: yq '.dependencies[] | select(.name == "bpmn-visualization") | .version' inst/htmlwidgets/bpmnVisualizationR.yaml
       - run: echo "OLD_VERSION=${{ steps.lookupOldBPMNVisuVersion.outputs.result }}" >> $GITHUB_ENV
-      - name: Delete old bpmn-visualization js lib file
+      - name: Delete the old file of bpmn-visualization
         run: rm inst/htmlwidgets/lib/bpmn-visualization/bpmn-visualization.min.js
-      - name: Download bpmn-visualization ${{ env.VERSION }}
+      - name: Download the ${{ env.VERSION }} file of bpmn-visualization
```

Comment:
Is it simpler?

```suggestion
      - name: Download the ${{ env.VERSION }} bpmn-visualization
```

Reply:
This would produce "Download the 0.29.1 bpmn-visualization" which is weird. It misses the "version" name at the end of the sentence.
Why not keeping the existing form "Download bpmn-visualization 0.29.1" or using the npm way for version declaration "Download bpmn-visualization@0.29.1"

Reply:
I thought `bpmn-visualization` and `0.29.1` was reversed in our messages, and I don't know why.
I find `bpmn-visualization 0.29.1` weird.

Good idea to switch to npm notation 😄

Reply:
```suggestion
      - name: Download bpmn-visualization@${{ env.VERSION }}
```